appeals/web/src/server/appeals/appeal-details/interested-party-comments/view-and-review/page-components/reject.mapper.js

```diff
@@ -25,27 +26,23 @@ export function rejectInterestedPartyCommentPage(appealDetails) {
 }
 
 /**
- * @param {Representation} comment - The comment containing the selected rejection reasons.
- * @param {RepresentationRejectionReason[]} rejectionReasonOptions - The available rejection reason options.
- * @returns {import('../../../../appeals.types.js').CheckboxItemParameter[]}
+ * @param {Representation} comment
+ * @param {RepresentationRejectionReason[]} rejectionReasonOptions
+ * @returns {import('../../../../../appeals/appeals.types.js').CheckboxItemParameter[]}
  */
 export function mapRejectionReasonOptionsToCheckboxItemParameters(comment, rejectionReasonOptions) {
-	const existingReasonIds = comment.rejectionReasons.map((reason) => reason.id);
+	const rejectionReasons = comment.rejectionReasons || [];
+	const rejectionReasonMap = new Map(rejectionReasons.map((reason) => [reason.id, reason]));
```

Comment:
nice!

Reply:
this file should have been removed, possibly bad rebase